src/app.js

```diff
@@ -47,6 +47,11 @@ function main() {
     generateButtonAnimation.goToAndPlay(0, true);
   });
 
+  const addPairToGeneratedSchedule = document.getElementById("add-pair-button");
+  addPairToGeneratedSchedule.addEventListener("click", function () {
+    console.log("addPairToGeneratedSchedule was click");
```

Comment:
Usually in production code, we'd want to omit all `console.log`s. You can create a TODO like below as a reminder.  

```suggestion
    // TODO: Add pair functionality
```